rules/cross-platform/guided_onborading_sample_rule.toml

```diff
@@ -0,0 +1,60 @@
+[metadata]
+creation_date = "2022/09/22"
+maturity = "production"
+updated_date = "2022/09/22"
+min_stack_comments = "Guided Onboarding will be available in Elastic 8.6+"
+min_stack_version = "8.6.0"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule helps you test and practice using alerts with Elastic Security as you get set up. It’s not a sign of threat
+activity.
+"""
+false_positives = [
+    """
+    This rule is not looking for threat activity. Disable the rule if you're already familiar with alerts.
+    """,
+]
+max_signals = 5
+interval = "24h"
+enabled = false
+from = "now-24h"
+index = [
+    "apm-*-transaction*",
+    "auditbeat-*",
+    "endgame-*",
+    "filebeat-*",
+    "logs-*",
+    "packetbeat-*",
+    "traces-apm*",
+    "winlogbeat-*",
+    "-*elastic-cloud-logs-*"
+]
+language = "kuery"
+license = "Elastic License v2"
+name = "My First Alert"
+note = """ 
+This is a test alert.
+
+This alert does not show threat activity. Elastic created this alert to help you understand how alerts work.
+
+For normal rules, the Investigation Guide will help analysts investigate alerts.
+
+This alert will show once every 24 hours for each host. It is safe to disable this rule.
```

Comment:
`max_signals` is set to 5 - should update to 1 to be accurate with this.

Also it will not be one per host, but one period across the environment (unless a threshold rule is used, though I think 1 total is better)

Reply:
or update it to reflect _up to 5 hosts_?